src/components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../css/Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <footer className="pageFooter">
+        <p>
+          Product of <br />
+          <a href="https://talentpath.com/">Talent Path</a>
+        </p>
+        <div>
+          <div>
+            <img  className="footerLogo" src="/images/SA_IconLogo-Final.png" alt="stonehavenLogo" />
```

Comment:
Same as above, kabob case.

```suggestion
            <img  className="footer-logo" src="/images/SA_IconLogo-Final.png" alt="stonehaven-logo" />
```